ci/env/install-miniconda.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+SCRIPT_DIR=$(builtin cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+WORKSPACE_DIR="${SCRIPT_DIR}/../.."
```

Comment:
and the `WORKSPACE_DIR` probably should be moved inside the `if [[ "${BASH_SOURCE[0]}" == "${0}" ]];` block